cmd/plugin/rpaasv2/cmd/log.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 tsuru authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	rpaasclient "github.com/tsuru/rpaas-operator/pkg/rpaas/client"
+	"github.com/urfave/cli/v2"
+)
+
+func NewCmdLogs() *cli.Command {
+	return &cli.Command{
+		Name:    "logs",
+		Usage:   "Fetches logs from a rpaasv2 instance pod",
+		Aliases: []string{"log"},
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "service",
+				Aliases: []string{"tsuru-service", "s"},
+				Usage:   "the Tsuru service name",
+			},
+			&cli.StringFlag{
+				Name:     "instance",
+				Aliases:  []string{"tsuru-service-instance", "i"},
+				Usage:    "the reverse proxy instance name",
+				Required: true,
+			},
+			&cli.StringFlag{
+				Name:     "pod",
+				Aliases:  []string{"p"},
+				Usage:    "specific pod to log from",
+				Required: false,
+			},
+			&cli.PathFlag{
+				Name:     "container",
+				Aliases:  []string{"c"},
+				Usage:    "specific container to log from",
+				Required: false,
+			},
+			&cli.IntFlag{
+				Name:     "lines",
+				Aliases:  []string{"l"},
+				Usage:    "number of earlier log lines to show",
+				Required: false,
+			},
+			&cli.IntFlag{
```

Comment:
Duration fits better.

```suggestion
			&cli.DurationFlag{
```